safe/impact_functions/earthquake/itb_earthquake_fatality_model/parameter_definitions.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+"""InaSAFE Disaster risk tool by Australian Aid - Parameter definition for
+Flood Vector on Building QGIS IF
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+
+from safe_extras.parameters.boolean_parameter import BooleanParameter
+from safe_extras.parameters.string_parameter import StringParameter
+from safe_extras.parameters.list_parameter import ListParameter
+from safe_extras.parameters.float_parameter import FloatParameter
+from safe_extras.parameters.dict_parameter import DictParameter
+
+
+def x_coefficient():
```

Comment:
I thought we agreed that eq params would be hard coded into the IF and not made available to the end user? (applied to all below).